### PR TITLE
Fix #51: better wording.

### LINKED ISSUE #51: "Found no Main Class." when users cancel the main class selection.
Steps:
1. Create more than one main class in the spring boot project. 
2. "Start" the app from boot dashboard. 
3. When you see the dropdown list of main classes, press ESC
4. The error message is not proper, actually, it finds more than one main classes but you cancel the selection.

### PR DESCRIPTION
**Pull request by Eskibear**
Add placeholder when specifying main class:
screen shot 2018-11-22 at 10 31 24 am

More proper warning message when users cancel selecting the main class. 
screen shot 2018-11-22 at 10 31 40 am

### CODE REVIEW

File: src/Controller.ts [hunk 1]
```diff
@@ -39,41 +39,43 @@ export class Controller {
...
+        const ok: boolean = await vscode.debug.startDebugging(
+            vscode.workspace.getWorkspaceFolder(vscode.Uri.parse(app.path)),
+            Object.assign({}, targetConfig, {
+                noDebug: !debug,
+                vmArgs
+            })
+        );
+        if (ok) {
+            // Cannot determine status. It always returns true now.
+            // See: https://github.com/Microsoft/vscode/issues/54214
```
Comment (andxu) [inline src/Controller.ts]: vscode has fixed this issue? while 
// Cannot determine status. It always returns true now.
            // See: https://github.com/Microsoft/vscode/issues/54214

Comment (Eskibear) [reply src/Controller.ts]: Yes, they have fixed it. But here we don't need this now. We use `onDidStartDebugSession` to determine which session is actually started.

File: src/Controller.ts [hunk 2]
```diff
@@ -39,41 +39,38 @@ export class Controller {
     public async startBootApp(app: BootApp, debug?: boolean): Promise<void> {
         const mainClasData = await vscode.window.withProgress(
             { location: vscode.ProgressLocation.Window, title: `Resolving main classes for ${app.name}...` },
-            () => { return this._getMainClass(app.path); }
+            () => { return this._getMainClass(app); }
         );
+        if (!mainClasData) {
+            vscode.window.showWarningMessage(mainClasData === null ?
```
Comment (andxu) [inline src/Controller.ts]: should not show warning if user cancels the selection.

Comment (Eskibear) [reply src/Controller.ts]: agree.

**Pull request merged**